metrics.py

```diff
@@ -464,6 +469,74 @@ def ingest_dashboard_revision_get():
 
     return None
 
+def ingest_attributes(api):
+    url = api.makeurl(['source', api.project, '_project', '_history'], {'meta': 1})
+    root = ET.parse(osc.core.http_GET(url))
+
+    points = []
+    count = 0
+    revision = 0
+    last_attribute_md5 = ''
+
+    last_values = {}
+
+    for rev in root.findall('./revision'):
+        revision = rev.get('rev')
+        time = datetime.utcfromtimestamp(float(rev.find('./time').text))
+
+        url = api.makeurl(['source', api.project, '_project'],
+                        {'meta': 1, 'rev': revision})
+        root = ET.parse(osc.core.http_GET(url))
+
+        attribute = root.find('.//entry[@name="_attribute"]')
+        if attribute is None:
+            continue
+
+        attribute_md5 = attribute.get('md5')
+        if attribute_md5 == last_attribute_md5:
+            continue
+        last_attribute_md5 = attribute_md5
+
+        url = api.makeurl(['source', api.project, '_project', '_attribute'],
+                                {'meta': 1, 'rev': revision})
+        root = ET.parse(osc.core.http_GET(url))
+        #print revision, time, ET.tostring(root)
+
+        for v in root.findall('.//attribute[@namespace="OSRT"]'):
+            attribute_name = v.get('name')
+            last_values.setdefault(attribute_name, None)
+            if last_values[attribute_name] == v.find('value').text:
+                continue
+            # TODO: no idea what fields to make
+            fields = {revision: revision }
+
+            points.append({
+                'measurement': 'attribute_osrt_{}'.format(attribute_name),
```

Comment:
The influxdb database in OSRT specific so no need to prefix measurements. Otherwise, all should be prefixed.